deploy/playbooks/deploy.yml

```diff
@@ -7,6 +7,7 @@
     - celery
   vars:
      app_name: "mita"
+     prod_fqdn: "mita.ceph.com"
```

Comment:
I didn't add it there because that would mean that to access your vagrant-deployed mita you would need to map mita.ceph.com to whatever IP your vagrant machine has